src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$cell-width: 100px;
+$cell-height: 100px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  width: calc(7 * $cell-width + 6 * 1px);
+  padding: 0 10px;
```

Comment:
To enhance readability, consider placing a space after the colon in property declarations. For example: `padding: 0 10px;`.